docs/vex.md

```diff
@@ -0,0 +1,234 @@
+  │  │  │ 
+  │  │  ├ CONTAINS PACKAGE pkg:apk/wolfi/glibc-locale-posix@2.36-r4?arch=x86_64
+  │  │  │  └ 🔗 12 Relationships
+  │  │  │ 
+  │  │  └ CONTAINS PACKAGE pkg:apk/wolfi/wolfi-baselayout@20221118-r0?arch=x86_64
+  │  │ 
+  │  └ GENERATED_FROM PACKAGE pkg:github/chainguard-images/images@bb50b31546d9ae34cbfb227f10024c757a7b1e22
+  │ 
+  └ 📄 DESCRIBES 0 Files
+
+```
+
+From the text diagram above, we know that the image contains 4 WolfiOS
+packages: 
+[`ca-certificates-bundle`](https://github.com/wolfi-dev/os/blob/main/ca-certificates.yaml),
+[`glibc-locale-posix`](https://github.com/wolfi-dev/os/blob/2925e13da033d01c2494ebda57a12c25b51dd13f/glibc.yaml#L415),
+[`tzdata`](https://github.com/wolfi-dev/os/blob/main/tzdata.yaml),
+and
+[`wolfi-baselayout`](https://github.com/wolfi-dev/os/blob/main/wolfi-baselayout.yaml).
+
+If we generate a VEX document from the image SBOM, `wolfictl` will capture all
+the history of impact analysis from the Wolfi Security Advisories into the
+VEX statements:
+
+```console
+wolfictl vex sbom cgr.dev/chainguard/static@sha256:c1c818750c4b36a994cd635df62a417123912e251ac53c9866e5794f2de3d073
+```
+
+```json
+{
+  "@context": "https://openvex.dev/ns",
+  "@id": "merged-vex-81361bfc0d69a7741cb9d3b12809f8b7678404701f19650c4c680477155de373",
+  "author": "Unknown Author",
+  "role": "Document Creator",
+  "timestamp": "2023-01-23T17:59:04.273286693-06:00",
+  "version": "1",
+  "statements": [
+    {
+      "vulnerability": "CVE-2022-39046",
+      "timestamp": "2023-01-23T17:59:04.273077241-06:00",
+      "products": [
+        "pkg:oci/static@sha256:c1c818750c4b36a994cd635df62a417123912e251ac53c9866e5794f2de3d073?arch=amd64&mediaType=application%2Fvnd.oci.image.manifest.v1%2Bjson&os=linux"
+      ],
+      "subcomponents": [
+        "pkg:apk/wolfi/glibc-locale-posix@2.36-r4"
+      ],
+      "status": "fixed"
+    }
+  ]
+}
+
+```
+
+Exploring the VEX document, we'll notice that `wolfictl` generated a document
+containing a single vex statement. The statement combines the required VEX
+fields from completeness: a vulnerability, a product, and an impact status:
+
+- It talks about the image, the “product” entry in the statement, identified by the image purl.
+- It pairs it with [CVE-2022-39046](https://nvd.nist.gov/vuln/detail/CVE-2022-39046),
+the identifier is in the vulnerability field.
+- And, finally, it also conveys an
+[impact status](https://github.com/openvex/spec/blob/main/OPENVEX-SPEC.md#status-labels):
+“fixed”.
+
+The document also specifies the subcomponent where the vulnerability originated:
+the Wolfi package identified by the purl `pkg:apk/wolfi/glibc-locale-posix@2.36-r4`.
+
+When evaluating the `static` image, a scanner can read the VEX metadata in the document 
+and turn off any false positives it finds by relying on data originated at
+the source: the Wolfi maintainers themselves. While scanners implement native
+filtering, [OpenVEX's `vexctl`](https://github.com/openvex/vexctl) can be used
```

Comment:
>While scanners implement native filtering

I think this phrase might be able to be misinterpreted as claiming that scanners implement native filtering _today_. I'd suggest something like:

>While scanners are preparing to support VEX filtering natively, ...